src/Build.UnitTests/NetTaskHost_E2E_Tests.cs

```diff
@@ -134,5 +134,26 @@ public void MSBuildTaskInNetHostTest()
             successTestTask.ShouldBeTrue();
             testTaskOutput.ShouldContain($"Hello TEST");
         }
+
+        [WindowsFullFrameworkOnlyFact]
+        public void NetTaskWithImplicitHostParamsTest()
+        {
+            using TestEnvironment env = TestEnvironment.Create(_output, setupDotnetEnvVars: true);
```

Comment:
testenvironment no longer has this flag - I removed it so we could see if the test logic still holds after refactorings in the vs18.0 branch